addons/web_editor/static/src/js/editor/odoo-editor/src/commands/enter.js

```diff
@@ -91,8 +91,9 @@ HTMLElement.prototype.oEnter = function (offset, firstSplit = true) {
  */
 HTMLHeadingElement.prototype.oEnter = function () {
     const newEl = HTMLElement.prototype.oEnter.call(this, ...arguments);
-    if (!newEl.textContent) {
+    if ([...newEl.textContent].every(char => char === '\u200B')) { // empty or all invisible
```

Comment:
Is there a particular reason to not use one of our invisible-related utils for that ?

Reply:
Yes, you had already asked and when I tried, none of them fit, that's what prompted [this PR](https://github.com/odoo/odoo/pull/114178) which is not merged yet.

Reply:
Oh yes indeed, I totally remember. 👀